docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -50,6 +50,14 @@ For information about how to support multiple tenants, see xref:security-openid-
 
 == Using the authorization code flow mechanism
 
+=== Configuring Quarkus to support authorization code flow
+
+To enable an authorization code flow authentication, the `quarkus.oidc.application-type` property must be set to `web-app`.
+Usually, the Quarkus OIDC ``web-app`` application type must be set when your Quarkus application is a frontend application which serves HTML pages and requires an OIDC single sign-on login.
```

Comment:
```suggestion
Usually, the Quarkus OIDC `web-app` application type must be set when your Quarkus application is a frontend application which serves HTML pages and requires an OIDC single sign-on login.
```

I added double quotes since the suggestion was in quotes, sorry...